docker-compose.yml

```diff
@@ -141,6 +141,10 @@ services:
       - _APP_STORAGE_DO_SPACES_SECRET
       - _APP_STORAGE_DO_SPACES_REGION
       - _APP_STORAGE_DO_SPACES_BUCKET
+      - _APP_STORAGE_BACKBLAZE_ACCESS_KEY
```

Comment:
Same comment as with the compose.phtml file.

Reply:
addressed